service-config/src/main/java/com/palantir/config/service/ServiceConfiguration.java

```diff
@@ -40,26 +40,23 @@
     public abstract Optional<SslConfiguration> security();
 
     /**
+     * Connect timeout for requests in ms.
+     */
+    public abstract Optional<Integer> connectTimeoutMs();
```

Comment:
I see. Yeah, that makes sense. ISO durations are too cryptic for most
people.

On Tue, Apr 26, 2016 at 4:43 PM Mark Elliot <notifications@github.com>
wrote:

> In
> service-config/src/main/java/com/palantir/config/service/ServiceConfiguration.java
> <https://github.com/palantir/http-remoting/pull/108#discussion_r61109936>:
>
> > @@ -40,26 +40,23 @@
> >      public abstract Optional<SslConfiguration> security();
> >
> >      /**
> > +     * Connect timeout for requests in ms.
> > +     */
> > +    public abstract Optional<Integer> connectTimeoutMs();
>
> I guess I was thinking of the Dropwizard Duration format, which might be
> useful to extract to a more general library for things like this, which
> allows encoding things like "100 milliseconds" and "100 ms" as well as "10
> seconds" or "2d" (for 2 days).
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/http-remoting/pull/108/files/082c00b9706d59959dc45a805b66db64cd9c36eb#r61109936>
>